<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSkeleton</class>
 <widget class="QWidget" name="EditSkeleton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>90</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit skeleton</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="FileChooser" name="model_file" native="true"/>
   </item>
   <item>
    <widget class="DCoordinate3GroupBox" name="model_offset" native="true"/>
   </item>
   <item>
    <widget class="DCoordinate3GroupBox" name="model_scale" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileChooser</class>
   <extends>QWidget</extends>
   <header>GUI/BasicWidgets/FileChooser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DCoordinate3GroupBox</class>
   <extends>QWidget</extends>
   <header location="global">GUI/BasicWidgets/DCoordinate3GroupBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
